README.md

```diff
@@ -63,6 +63,46 @@ You can add a thumbnail to a post; simply link the thumbnail in the front
 matter in the markdown file for the post. The thumbnail should be a 150px
 square for best results.
 
+### Calendar Events
+
+The calendar events are accessed from the public CSS google calendar also
+available [here][calendar]. They are accessed from the calendar using
+[FullCalendar][fullcalendar] and an API key managed from the Google Cloud
+Console.
+
+To add a new calendar event, a member of the CSS committee with access to the
+calendar must add a new event, and it will automatically appear on the
+website.
+
+#### Event formatting
+
+The event data displayed on the webiste is taken directly from the google
+calendar event. Some properties are extracted using XML-like tags from the
+event description - note that this does **not** imply that other XML tags are
+implemented, so no other tags aside from the ones specified here should be
+used.
+
+| Content       | Source                                                                                                  |
+| :------------ | :------------------------------------------------------------------------------------------------------ |
+| title         | the name of it appears on the calendar                                                                  |
+| date          | the date and time as it appears on the calendar                                                         |
+| location      | the location as it appears on the calendar                                                              |
+| description   | `<desc></desc>` tags in the event description, otherwise, the full event description                    |
+| facebook link | `<fb></fb>` tags in the event description, otherwise, the first `<a></a>` tags in the event description |
```

Comment:
Not quite, for the link to be accessed from the "Facebook Event" button, it must be in the `<fb>` tag. The `<a>` tags are only removed if they match in the `<fb>` tags. 

i.e. for a link to be put into the button, it must be within the `<fb>` tag. The `<a>` tag is optional - caused by hyperlinks in the Google Calendar event

Reply:
Perhaps it wasn't quite clear enough in the code

Reply:
Ahhh alright, thanks for the clarification, will fix that.